README.md

```diff
@@ -18,8 +18,21 @@ Add spree_ams to your Spree store's Gemfile:
 gem 'spree_ams', :github => 'hhff/spree_ams', :branch => '2-3-alpha'
 ```
 
+By default spree ember will use relative urls for the spree product images. 
+
+If server serving the assets is somewhere other than the same place as your ember app
+
+make sure `config.action_controller.asset_host` is set in your environment config
```

Comment:
once this is done, can you please squash this PR to a single commit?  then I'll merge and move over to 2-3